presto-orc/src/main/java/com/facebook/presto/orc/reader/LongDictionaryBatchStreamReader.java

```diff
@@ -146,8 +146,8 @@ public Block readBlock()
             // Data has nulls
             if (dataStream == null) {
                 // The only valid case for dataStream is null when data has nulls is that all values are nulls.
-                int nullValues = presentStream.getUnsetBits(nextBatchSize);
-                if (nullValues != nextBatchSize) {
+                int notNullValues = presentStream.countBitsSet(nextBatchSize);
```

Comment:
This change doesn't seem to be related?

Reply:
@rongrong this change was in the original commit "Improve ORC boolean reader". That commit removed `public int getUnsetBits(int batchSize)` from BooleanInputStream. Since this is cherry-pick, should we maintain the original commit structure?